src/interface/common.thrift

```diff
@@ -100,4 +102,10 @@ struct Pair {
     2: string value,
 }
 
+struct IndexItem {
+     1: required IndexID          index_id,
+     2: required i32              tagOrEdge,
```

Comment:
By the way,   why we need tagOrEdge here?  index_id is not enough?

And where did we use the IndexItem?

Reply:
Different index id may correspond to the same tagID.
IndexItem will be used for storage layer and graph lookup .

Reply:
> By the way, why we need tagOrEdge here? index_id is not enough?
> 
> And where did we use the IndexItem?

I need a detailed explanation：
When addVertices or addEdges start executing, maybe have multiple kinds of tags or edgeType in the same request. And a tag or edgeType may have multiple indexes.Therefore, I need to know the relationship between specific tags and indexes.

Also, IndexItem will be used for each vertex or edge write, for example insert|delete|update ，the IndexId and tagOrEdge is required.

Reply:
Make sense.